### ISSUE TITLE
New Problem: Calculate BLEU Score (NLP)

### ISSUE THREAD

**Issue by rittik9** [Open-Deep-ML/DML-OpenProblem#192]
<img width="1043" alt="MZygZ" src="https://github.com/user-attachments/assets/9d75e390-e3e2-48e7-82a9-d4fa2aad4742">

**Comment by saitiger** [comment]
Following up on this, the implementation of METEOR can also be done. If approved, I can work on this.
![Screenshot 2024-11-23 120136](https://github.com/user-attachments/assets/5dda001d-dbf4-4397-b8fb-fe57ec5b985b)

**Comment by rittik9** [comment]
I think it'll be better if you open a new issue for meteor.

**Comment by moe18** [comment]
I agree try and keep the new problems separate, I think both problems would be a great addition

**Comment by saitiger** [comment]
Alright I'll have open a separate PR for it once I am done

**Comment by saitiger** [comment]
@moe18 I am running into some issues with the implementation. Will submit the PR asap, most probably by Monday night.
